src/lib/bridge/handlers/fetch_recommended_price.ts

```diff
@@ -0,0 +1,97 @@
+import {SimpleHandler} from './main';
+import {RequestType} from './types';
+import {environment} from '../../../environment';
+import MessageSender = chrome.runtime.MessageSender;
+
+export interface CSFloatListing {
```

Comment:
This type exists as `Contract` in the `float_market.ts` file. The term `Contract` and `Listing` are interchangeable, but internally they are referred to as `Contracts`